src/models/order.js

```diff
@@ -29,6 +29,6 @@ export default class Order extends EasyPostObject {
   lowestRate(carriers, services) {
     const rates = this.rates || [];
 
-    return Utils.getLowestRate(rates, carriers, services);
+    return new EasyPostClient('fake_api_key').Utils.getLowestRate(rates, carriers, services);
```

Comment:
issue: we should not be constructing clients inside our own code. The client is for the user. Because we have direct access to the code before it gets compiled, we should still be able to reference the utils relatively. It's our users who cannot do this because they only get access to the single exposed client.

Reply:
Well, that won't work then, because we don't want to create a new client like Justin said, but we also don't have access to an existing client inside a model.

Do we want to keep these instance functions? `myOrder.lowestRate()` versus `myClient.Utils.lowestRate(myOrder.rates)`?

Reply:
We should keep them instance function for the object, otherwise it would be a breaking change